websocket/connection.go

```diff
@@ -49,6 +66,19 @@ func (c *GorillaConn) Write(p []byte) (int, error) {
 	return len(p), nil
 }
 
+// SetDeadline sets both read and write deadlines, as per net.Conn interface docs:
+// "It is equivalent to calling both SetReadDeadline and SetWriteDeadline."
+// Note there is no synchronization here, but the gorilla implementation isn't thread safe anyway
+func (c *GorillaConn) SetDeadline(t time.Time) error {
```

Comment:
Is this to make `GorillaConn` implement `net.Conn` interface? Was it not implemented properly before? Did it not matter until now?

Reply:
Yes it is; I am not sure being a full net.Conn was ever a goal or if you just wanted it to be an `io.ReadWriter` but it seems a shame not to add a the simple interface to make it a net.Conn